tests/functional/sync_force_auth.js

```diff
@@ -46,22 +47,16 @@ define([
       return this.remote
         .then(setupTest(this, true))
 
-        // add a slight delay to ensure the page does not transition
-        .sleep(1000)
-
-        // the page does not transition.
-        .then(testElementExists('#fxa-force-auth-header'))
-        .then(testIsBrowserNotified(this, 'can_link_account'))
-        .then(testIsBrowserNotified(this, 'login'));
+        // page does not transition
+        .sleep(2000)
```

Comment:
Was there a problem with `1000` or is this more an act of prudence? Is `2000` definitely enough for all cases?